web/src/features/hfiCalculator/components/DailyViewTable.tsx

```diff
@@ -326,9 +338,14 @@ export const DailyViewTable = (props: Props): JSX.Element => {
                         .sort((a, b) => (a[1].code < b[1].code ? -1 : 1))
                         .map(([stationCode, station]) => {
                           const daily = getDailiesByStationCode(
-                            1,
+                            numPrepDays,
                             props.dailies,
                             station.code
+                          ).filter(
```

Comment:
I know there's some duplication here - but it's all dead code that the Conor PR should replace.